tests/unit/s2n_record_read_test.c

```diff
@@ -20,6 +20,8 @@
 #include "utils/s2n_safety.h"
 
 #define SSLV2_MIN_SIZE 3
+/* Record Header length for TLS1.3 and TLS1.2 protocols */
+const int S2N_TLS_RECORD_HEADER_LEN = 5;
```

Comment:
Don't redefine this: https://github.com/aws/s2n-tls/blob/ed58a00ffbc14bf9470387bbb3e65c1a289efdd4/tls/s2n_record.h#L31

Reply:
Awesome, I was looking for this